docs/user-guide/glossary/edit-glossaries.md

```diff
@@ -1,6 +1,28 @@
 ### Prerequisite
 See [glossary roles and permission](/user-guide/glossary/glossary-roles-permissions) for permission setup.
 
+### From Glossary page
+1. Click `Glossary` menu.
+<figure>
+![Glossary page](/images/glossary-page.png)
+</figure>
+1. Go to entry you wish to edit, fill in part of speech or description.
```

Comment:
A few places skipping "the", but I think we can afford to use full sentences here.